### PR TITLE
Fix logic when deciding if apt-get package should be installed

### PR DESCRIPTION
**Pull request by manandbytes**
Install apt-get package only if it's not installed.

### CODE REVIEW

**Comment by manandbytes** [comment]
Sorry, no tests this time ;-)

File: methods/el-get-apt-get.el [hunk 1]
```diff
@@ -123,7 +123,7 @@ password prompt."
 (defun el-get-apt-get-install-if-needed (package url post-install-fun)
   "Call `el-get-apt-get-install' if PACKAGE isn't installed yet.
 The installation status is retrieved from the system, not el-get."
-  (when (el-get-dpkg-package-installed-p (or (plist-get (el-get-package-def package) :pkgname)
+  (unless (el-get-dpkg-package-installed-p (or (plist-get (el-get-package-def package) :pkgname)
 					     (el-get-as-string package)))
```
Comment (npostavs) [inline methods/el-get-apt-get.el]: The indentation looks wrong on line 127.

**Comment by manandbytes** [comment]
Any updates?

File: methods/el-get-apt-get.el [hunk 2]
```diff
@@ -123,8 +123,8 @@ password prompt."
 (defun el-get-apt-get-install-if-needed (package url post-install-fun)
   "Call `el-get-apt-get-install' if PACKAGE isn't installed yet.
 The installation status is retrieved from the system, not el-get."
-  (when (el-get-dpkg-package-installed-p (or (plist-get (el-get-package-def package) :pkgname)
-					     (el-get-as-string package)))
+  (unless (el-get-dpkg-package-installed-p (or (plist-get (el-get-package-def package) :pkgname)
+                                             (el-get-as-string package)))
```
Comment (npostavs) [inline methods/el-get-apt-get.el]: Indentation still looks wrong, should `(el-get-as-string...)` should line up with `(plist-get...`

**Comment by manandbytes** [comment]
Seems it's ok, finally.

**Comment by npostavs** [comment]
Yup :) I should see if it's possible to automate the indent check. We have the `cleanup-whitespace.sh` script, so it should just be matter of using it...

**Pull request merged**